src/amber/cli/templates/app/shard.yml.ecr

```diff
@@ -33,6 +33,10 @@ dependencies:
     github: amberframework/jasper-helpers
     version: ~> 0.2.5
 
+  amber_render_module:
```

Comment:
This goes against the standard layout that the generators currently use.  Either we go one way or the other. I will raise this request to the core team and provide feedback.  If approved, we will rewrite the current generators to use the module approach.  If not, we will stick with the current approach and request that the plugins comply.

Reply:
I personally don't see why the two formats cannot co-exist.  They have done so in my Amber applications from the outset.  The argument to remain with the existing layout is that people coming from a Rails environment will feel comfortable with the similarity in the layout.  The argument for modular layout is better code management, Principle of Proximity etc.  From my experience when working on large Rails projects the lack of modular structure and Principle of Proximity is very frustrating, particularly when coming into a project as a contractor with a very large codebase.

I don't think we need to throw the baby out with the bath water.  We could quite easily make the code layout an option with `amber new` so the developer could chose to create an application with the standard layout or chose a modular layout.  We don't even need to have 2 versions of the templates in the amber cli source.  We can use recipes to create an app either from a standard recipe or from a modular recipe, e.g
```
$ amber new myapp  # create an app with the standard layout 
$ amber new -m myapp  # create an app with the modular layout - from a pre-defined recipe
```